app/Config/View.php

```diff
@@ -41,4 +41,15 @@ class View extends BaseView
      * @var array
      */
     public $plugins = [];
+
+    /**
+     * View Decorators are class methods that will be run in sequence to
+     * have a chance to alter the generated output just prior to caching
+     * the results.
+     *
+     * All classes must implement CodeIgniter\View\ViewDecorator
+     *
+     * @var array
```

Comment:
The current class is actually `CodeIgniter\View\Interfaces\ViewDecorator` but my suggestion will assume we go with the rename below. Regardless this should use the `class-string` prototype.
```suggestion
     * @var class-string<CodeIgniter\View\ViewDecoratorInterface>[]
```

Reply:
I'll change it but what's the purpose of labeling it like this?

Reply:
Yep! It tells static analysis "this can only contain strings that are actually class names of this type". Pretty handy actually.